### DISCUSSION TITLE
Define convenience URLs

### DISCUSSION THREAD

**Discussion by mil-ad** [AdguardTeam/AdGuardHome#5275] [Q&A]
I was wondering if I can use my adguard home to define some URLs that resolve to my various web services running on my local network? Something like:

`plex.home` (or even `plex` on its own?) -> `192.168.1.100:8383`
`syncthing.home` -> `192.168.1.50:8381`

I noticed there's "DNS Rewrite" but it doesn't seem to support this.

**Comment by D13410N3** [comment]
This thing can't be done via DNS.
If you'd like to use URL like "http://plex.home" that forwards you to "http://192.168.1.100:8383" you should use something like [nginx reverse proxy](https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/)

**Reply by mil-ad** [reply]
interesting. I already have a couple of reverse proxies that respond to real URLs. For example:

some_domain_i_own.com -> 192.168.1.100:8383

I can make the proxy react to `plex.home` but I still need to modify the DNS to forward `plex.home` to my reverse proxy right?

**Reply by D13410N3** [reply]
Yes. Use "Filters -> DNS rewrites" in AdGuard Home
